docs/BLUETOOTH.md

```diff
@@ -3,17 +3,17 @@
 Bluetooth support can be enabled by connecting a USB Bluetooth dongle to your AmpliPi.
 
 ## Hardware support
-AmpliPi currently supports at most one Bluetooth stream at a time. If a Bluetooth dongle is connected when the AmpliPi is started a Bluetooth stream is created automatically. 
+AmpliPi currently supports at most one Bluetooth stream at a time. If a Bluetooth dongle is connected when the AmpliPi is started a Bluetooth stream is created automatically.
 
 ### Tested Hardware
--  [this Bluetooth dongle](https://www.amazon.com/gp/product/B08LVH5BCP/).
+-  [ZEXMTE Long Range USB Bluetooth Adapter](https://www.amazon.com/gp/product/B08LVH5BCP/).
```

Comment:
Updated this to be accessible in a printed format, but should we really be advertising an unaffiliated product like this? as much as it has worked, it may be unwise to seemingly endorse their products as a whole.

Reply:
One strategy here would be to buy a selection of different BT chips and list the ones that were tested and whether or not they worked. It might be helpful to also note what underlying IC and linux driver each is using.

At the moment we have only tested one USB BT module.

Reply:
"as much as it worked, it may be unwise to seemingly endorse their products"
I can see this either way; it's extremely helpful for folks to have an example of something that has worked, but it definitely does give the impression we "support" this. (i mean... we kinda do, don't we? :p) 

I can see when #610 lands, we define specific streams/plugins as "supported" or "community" - and if I recall correctly, the Bluetooth stuff was explicitly donated from the community. This might be the correct level of investment - we definitely want to opportunistically support these, but also be careful what streams we explicitly support and prioritize fixes for with our smaller workforce.

My vote would be to leave your re-written reference in for now and revisit when we split out streams.